mycroft/filesystem/__init__.py

```diff
@@ -39,7 +39,7 @@ def __init_path(path):
 
         # Migrate from the old location if it still exists
         # TODO: remove in 22.02
-        if isdir(old_path):
+        if not isdir(path) and isdir(old_path):
             if isdir(path):
```

Comment:
This can never be true now so L43, L44 can be removed

Reply:
Not sure that I follow...
> If the new path doesn't exist but the old path does exist

Reply:
oh the next lines right - this change is saying never remove the new directory :+1: